tools/integration_tests/tokenserver/test_support.py

```diff
@@ -45,15 +45,6 @@ def setUp(self):
         cursor = self._execute_sql(('DELETE FROM nodes'), ())
         cursor.close()
 
-        cursor = self._execute_sql(('DELETE FROM services'), ())
-        cursor.close()
-
-        # Ensure the necessary services exists in the db.
-        self._add_service('sync-1.1', '{node}/1.1/{uid}',
```

Comment:
As of right now, the database migrations are responsible for populating the services table with two services, so I updated the tests to reflect that. This may change in the future, but I wanted everything to at least be consistent for now.